src/domains/wallet/pages/WalletDetails/components/WalletHeader/WalletHeader.tsx

```diff
@@ -252,7 +252,7 @@ export const WalletHeader = ({
 						<Avatar size="lg" address={wallet.address()} shadowClassName="ring-theme-secondary-900" />
 					</div>
 
-					<div className="flex flex-col w-full overflow-hidden">
+					<div className="flex flex-col w-full py-2 -my-2 pr-2 -mr-2 overflow-hidden">
```

Comment:
`-mr-2` doesn't work because we're using `space-x-4` :hide:
initial width is pretty simple :trollbounce: